action.sh

```diff
@@ -1 +1,63 @@
 #!/bin/bash
+
+set -euo pipefail ## if anything in the script fails => exit 
+
+REMOTE_USER="root"
+REMOTE_IP="68.183.145.114"
```

Comment:
let's use variable/argument instead of hardcoded value.

```
REMOTE_USER="root"
REMOTE_IP="68.183.145.114"
```

for example: --runs-on root@68.183.145.11